tests/test_datasets.py

```diff
@@ -10,6 +10,11 @@ def setUp(self) -> None:
     def test_create_env(self):
         env = SpaceEnv(self.dataset_name)
         assert isinstance(env, SpaceEnv)
+
+    def test_all_envs(self):
+        for dataset in DataMaster().list_datasets():
```

Comment:
I don't think it's a good practice to download the datasets for testing, the server response should be mocked or something

Reply:
Ok I will comment and leave a note.